src/util/notifications_util.js

```diff
@@ -492,10 +498,63 @@ function get_notification_logger(locking, namespace, poll_interval) {
     });
 }
 
+/**
+ * add_connect_file Creates a new connection file from the given source file.
+ * If source has an auth field in it's request_options_object, it is encrypted.
+ * @param {Object} target_content  cli parameters with the from_file as source file
```

Comment:
I'm not sure that in this function context, it is clear what is "from_file as source file".